.github/workflows/bitcoin-tests.yml

```
@@ -146,7 +146,8 @@ jobs:
           - tests::signer::v0::single_miner_empty_sortition
           - tests::signer::v0::multiple_miners_empty_sortition
           - tests::signer::v0::block_proposal_timeout
-          - tests::signer::v0::rejected_blocks_count_towards_miner_validity
+          - tests::signer::v0::reorg_attempts_count_towards_miner_validity
+          - tests::signer::v0::late_reorg_attempts_do_not_count_towards_miner_validity
```

I think this should be:
```suggestion
          - tests::signer::v0::reorg_attempts_activity_timeout_exceeded
```